admin.list

```diff
@@ -0,0 +1,32 @@
+scripts/2.6.0: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/3.0.0: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/3.2.1: Do not load a specific version of the scripts module. Only do: module load scripts
+
+scripts/4.0.0: Do not load a specific version of the scripts module. Only do: module load scripts
+
+VSC-tools/0.1.3-ictce-4.1.13-scoop: Do not use the VSC-tools module anymore. Load vsc-mympirun directly.
+
+VSC-tools/0.1.5-ictce-4.1.13-scoop: Do not use the VSC-tools module anymore. Load vsc-mympirun directly.
```

Comment:
People are going to wonder why they need to switch to `vsc-mympirun`, we might as well explain it once in the userwiki and point them to it, rather than answer N mails about it...